qemu/tests/qemu_guest_agent.py

```diff
@@ -235,7 +235,10 @@ def gagent_start(self, session, vm):
         :param vm: Virtual machine object.
         """
         error_context.context("Try to start qemu-ga service.", LOG_JOB.info)
-        s, o = session.cmd_status_output(self.params["gagent_start_cmd"])
+        if self.params.get("gagent_serial_type") == 'isa':
+            s, o = session.cmd_status_output(self.params["gagent_start_cmd"], safe=True)
+        else:
+            s, o = session.cmd_status_output(self.params["gagent_start_cmd"])
```

Comment:
Any side effect for use safe mode for all ?    IMO, it does not effect others situation to get the expected prompt pattern.

Reply:
okay, changed.

Reply:
@yanan-fu  (1/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.isa_serial.gagent_check_log.q35: STARTED
 (1/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.isa_serial.gagent_check_log.q35: PASS (91.26 s)
 (2/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_guest_agent.isa_serial.gagent_check_log.q35: STARTED
 (2/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_guest_agent.isa_serial.gagent_check_log.q35: PASS (100.67 s)